_posts/2024-08-27-How-does-the-rank-normalization-work-in-hybrid-search.md

```diff
@@ -0,0 +1,507 @@
+---
+layout: post
+title:  "An Overview of Rank Normalization in Hybrid Search"
+authors:
+ - ljeanniot
+date: 2024-08-30
+categories:
+ - technical-post
+meta_keywords: semantic search engine, neural search engine, keyword and natural language search, hybrid search.
```

Comment:
Please update the meta with the following: 
meta_keywords:  rank normalization, hybrid search, OpenSearch score normalization, search relevance, user experience

meta_description:  Learn about rank normalization in hybrid search and how to OpenSearch can help you standardize scores from diverse algorithms to deliver more accurate and fair search results.